provisioning/osc-provision

```diff
@@ -201,6 +211,20 @@ provision_nodes() {
   $command || error_out "Node Provision failed. See log at ${LOGFILE}." ${ERROR_CODE_PROVISION_FAILURE}
 }
 
+sync_files() {
+  # Sync files from variable with format "file1:dest1 file2:dest2"
+  files=$1
+  nodes=$2
+
+  for node in $nodes; do
+    for file in $files; do
+      source=${file%:*}
+      dest=${file#*:}
+      ${SCP_CMD} ${source} root@${node}:${dest}
```

Comment:
Would be nice to support URLs for source - i.e.: in the case of the IPA CA, we could just have it pull directly from the IPA.